dw-al2-base-ami/dw-general-ami-install.sh

```diff
@@ -3,13 +3,13 @@ set -eEu
 
 ## Script to prepare general Dataworks AMI
 
-echo "HTTP_PROXY=$HTTP_PROXY"
-echo "HTTPS_PROXY=$HTTPS_PROXY"
+echo "HTTP_PROXY=$http_proxy"
+echo "HTTPS_PROXY=$https_proxy"
```

Comment:
```suggestion
echo "HTTP_PROXY=$HTTP_PROXY"
echo "HTTPS_PROXY=$HTTPS_PROXY"
```

```suggestion
echo "HTTP_PROXY=$HTTP_PROXY"
echo "HTTPS_PROXY=$HTTPS_PROXY"
```

These are debug lines, so should be stating what the correctly-cased variables are set to, otherwise confusion may  occur when HTTPS_PROXY is set to something different to https_proxy on its way into the script, but this debug statement would have you believe they're set to the same value.